initial_pose: 
  x: 0
  y: 1
  z: 1
  R_deg: 0
  P_deg: 0
  Y_deg: 0
  camera_deg: -90
